addons/html_builder/static/src/builder/options/alignment_option.xml

```diff
@@ -11,4 +11,16 @@
     </BuilderRow>
 </t>
 
+<t t-name="html_builder.VerticalAlignmentOption">
+    <!-- TODO add title tooltip row Vertical Alignment-->
+    <BuilderRow label.translate="Vert. Alignment" level='level'>
+        <BuilderButtonGroup action="'setVerticalAlignment'">
+            <BuilderButton title.translate="Align Top" actionParam="'align-items-start'" iconImg="'/website/static/src/img/snippets_options/align_top.svg'"/>
```

Comment:
Not sure if you want to import the icon in html_builder (the code will have to be in website anyway after)

Reply:
Yes, we'll have to check all the icons that are website or general.